### PR TITLE
Update readme with reference to request and FAQ

### PR DESCRIPTION
**Pull request by naugtur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # xhr
 
-A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/).
+A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/), [webpack](https://webpack.github.io/) etc.
+
+API is a subset of [request](https://github.com/request/request) so that the same code can be used in the browser and Node.js.
```
Comment (yoshuawuyts) [inline README.md]: I'd mention that the streaming API has been removed; size reasons and stuff :sparkles:

Comment (naugtur) [inline README.md]: I wouldn't phrase it like that, but yes, there is no streaming API as it's not yet possible in the browser in general

Comment (yoshuawuyts) [inline README.md]: cool, keen to see how you'll phrase it then :grin:

Comment (naugtur) [inline README.md]: I pushed it, but it's almost midnight here, so I'm not sure if I did well :)

**Pull request merged**